.github/workflows/update-helm-charts-index.yml

```
@@ -16,7 +16,7 @@ jobs:
         run: |-
           export TAG=${{ github.ref_name }}
           git_tag=$(echo "${TAG#v}")
-          chart_tag=$(yq r Chart.yaml version)
+          chart_tag=$(yq '.version' Chart.yaml)
```

Might want to use the raw string, instead of the default JSON?

```suggestion
          chart_tag=$(yq -r '.version' Chart.yaml)
```

---

Good call 👍  I've added this while resolving the merge conflict.